### ISSUE TITLE
Filterlabeling issue

### ISSUE THREAD

**Issue by Testato** [collin80/SavvyCAN#544]
You agree it's a Savvycan issue ?
https://github.com/cantools/cantools/issues/505

**Comment by collin80** [comment]
Yes, it seems I've managed to output a boolean value in an odd way. I guess the parser in savvycan can load it fine but it really needs to be corrected.
